src/main/java/org/broadinstitute/hellbender/tools/exome/pulldown/BayesianHetPulldownCalculator.java

```diff
@@ -184,6 +184,20 @@ private double getSingletonLogLikelihood(final Map<Nucleotide, List<BaseQuality>
     }
 
     /**
+     * Checks of a given base is in BASES
+     * @param base a nucleotide
+     * @return boolean
+     */
+    private boolean isProperBase(final Nucleotide base) {
+        for (final Nucleotide properBase : BASES) {
+            if (base.equals(properBase)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
+    /**
      * Returns map of base-pair to error probabilities at a given locus. All reads are considered (not just ACTG)
```

Comment:
It is still applicable. The improper base calls, however, will be ignored in the inference.